src/main/scala/scalismo/faces/image/pyramid/GaussPyramid.scala

```diff
@@ -68,14 +59,24 @@ object GaussPyramid {
   /**
     * Standard reduce operation dividing the image size along each dimension by 2.
     */
-  def reduce[A: ClassTag](implicit ops: ColorSpaceOperations[A]) = new ImageFilter[A, A] {
+  def reduce[A: ClassTag](implicit ops: ColorSpaceOperations[A]) = reduceScaled[A](0.5)
+
+  /**
+    * Returns an image filter that reduces an image according to a scaling factor.
+    * @param scale 1.0 > scale > 0
+    */
+  def reduceScaled[A: ClassTag](scale: Double)(implicit ops: ColorSpaceOperations[A]) = new ImageFilter[A, A] {
+    require( scale < 1.0 && scale > 0, "scale must be on (0,1.0)" )
+
     import ColorSpaceOperations.implicits._
+
+    private val invScale = 1.0/scale
+
     override def filter(img: PixelImage[A]): PixelImage[A] = {
-      val w = img.width / 2
-      val h = img.height / 2
-      val filteredImage = img.withAccessMode(AccessMode.MirroredPositionFunctional((a:A, b:A)=>2*:a-b)).filter(GaussPyramid.filter)
-      val interpolatedImage = filteredImage.interpolate
-      PixelImage[A](w, h, (x: Int, y: Int) => filteredImage(x * 2 + 1, y * 2 + 1))
+      val w = (img.width * scale).toInt
+      val h = (img.height * scale).toInt
+      val filteredImage = img.withAccessMode(AccessMode.MirroredPositionFunctional((a: A, b: A) => 2 *: a - b)).filter(GaussPyramid.filter)
+      PixelImage[A](w, h, (x: Int, y: Int) => filteredImage((x * invScale).toInt + 1, (y * invScale).toInt + 1))
```

Comment:
Here, we still have the issue of integer relations (see previous comment). I see multiple possibilities to resolve this:
- force integer relations
- use a consistent way of resampling: `resample` (as you did for upscaling in `LaplacePyramid` anyway), it works for any scaling factor
- let it be, file an issue and add a comment to be careful with fractional values
- show some evidence that it does not really matter in practice